loadtest/store/memstore/store.go

```diff
@@ -193,6 +195,21 @@ func (s *MemStore) SetEmojis(emoji []*model.Emoji) error {
 	return nil
 }
 
+func (s *MemStore) SetReaction(reaction *model.Reaction) error {
+	if s.reactions[reaction.PostId] == nil {
+		s.reactions[reaction.PostId] = map[string]map[string]*model.Reaction{}
+		s.reactions[reaction.PostId][reaction.EmojiName] = map[string]*model.Reaction{}
+	} else if s.reactions[reaction.PostId][reaction.EmojiName] == nil {
+		s.reactions[reaction.PostId][reaction.EmojiName] = map[string]*model.Reaction{}
+	}
+	s.reactions[reaction.PostId][reaction.EmojiName][reaction.UserId] = reaction
+	return nil
+}
```

Comment:
This looks correct but I am not sure we need to store as much information. I think that for now we should be okay with just mapping posts to reactions, so given a `PostId` we would get a slice of all reactions for that post. This would simplify a lot the code above.